pkg/kfapp/ossm/ossm_installer.go

```diff
@@ -152,6 +164,48 @@ func (ossm *Ossm) createConfigMap(cfgMapName string, data map[string]string) err
 	return nil
 }
 
+func (ossm *Ossm) migrateDSProjects() error {
+
+	client, err := clientset.NewForConfig(ossm.config)
+	if err != nil {
+		return err
+	}
+
+	// Define the selector to find the namespaces we are interested in
+	selector := labels.SelectorFromSet(labels.Set{"opendatahub.io/dashboard": "true"})
+
+	// List the namespaces with the specific label
+	namespaces, err := client.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{LabelSelector: selector.String()})
+	if err != nil {
+		return fmt.Errorf("failed to get namespaces: %v", err)
+	}
+
+	var errs []string
```

Comment:
I would rather think about keeping `error` structs instead. We have a few libraries from `hashicorp` already in place, so maybe we could use this https://github.com/hashicorp/go-multierror